setting_up_firely_server/configuration/appsettings.rst

```diff
@@ -683,7 +683,7 @@ are used for your Firely Server, by changing the ``PipelineOptions``.
           // etc.
         ],
         "Exclude": [
-          "Vonk.Core.Operations"
+          "Vonk.Plugin.Operations"
```

Comment:
Hi @alexzautke, do you know the reason why we need to exclude `Vonk.Plugin.Operations`, seems working without exclude it. See Louis' change in PR https://github.com/FirelyTeam/Vonk.Infrastructure/pull/182/files, it passes the nightly integration tests (https://dev.azure.com/firely/vonk/_build/results?buildId=37539&view=results).

I mean, do we still need to suggest a user to exclude it?

Reply:
Otherwise duplicated services would be available on the admin pipeline. The user needs to use Vonk.Administration.Api.Operations. All other operations don't work on the admin pipeline.

Reply:
Thanks @alexzautke for the explanation. I have updated `Vonk.Facade.Starter` to make them aligned. https://github.com/FirelyTeam/Vonk.Facade.Starter/pull/78

Reply:
Good point, I missed that yesterday myself!